### DISCUSSION TITLE
Getting "Econn Refused" from data proxy

### DISCUSSION THREAD

**Discussion by bingomanatee** [prisma/prisma#14524] [Q&A]
52.205.145.236:443
 --- is refusing connections; nothing in the data proxy logs for this.

**Comment by nurul3101** [comment]
Hey @bingomanatee 👋 
Are you getting this error in your application logs or while using Data Browser or Query Console on [Prisma Data Platform](https://cloud.prisma.io/)?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
